src/main/scala/http4s/extend/Algebra.scala

```diff
@@ -0,0 +1,6 @@
+package http4s.extend
+
+object Algebra {
+
+  final class ThrowableCompleteMessage(val message: String) extends AnyVal
```

Comment:
I'm not sure if this is the best name for it, I couldn't come up with a better name though :(

Reply:
very ugly indeed. I will look for something else